index.html

```diff
@@ -5,11 +5,22 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Vite App</title>
+  <meta name="description" content="Play incredible Star Wars Quiz! Can you beat the others? CLICK TO PLAY!">
+  <title>Star Wars Quiz</title>
+  <link rel="stylesheet" href="styles/index.css" />
 </head>
 
 <body>
-  <div id="app"></div>
+  <div id="app">
```

Comment:
Another wrapper or some styling for `#app` might be needed to prevent the content from dispersing too much on wider screens by setting some `max-width`. I've had in mind screens wider than 1440px.

Reply:
I changed this by moving 'bg-image' class to div with id 'app'. And set the width to 1440px in child div.